Buffers/BufferIn.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import numpy as np
+testData = np.array([[1,2,3],[4,5,6]],dtype=np.uint8)
+def createBufferIn(data):
+    buffer = np.unpackbits(data).flatten()
+    print(buffer)
+    countIn = 8 #Na przykład 8, wielkość ramki bez nagłówków
+    offsetIn = 0
+    for offsetIn in range (0, len(buffer)-1, countIn): #Pętla opróżniająca cały buffer
+            dataIn = np.frombuffer(buffer, dtype='uint8', count=countIn,offset=offsetIn)
+            if len(dataIn) % 2 != 0:
+                raise Exception('Data is uneven!')
+                returnVal = []
+                return returnVal
```

Comment:
jesteś pewny, ze chcesz za każdym razem sprawdzać ten warunek? każdorazowe sprawdzanie kosztuje czas procesora, jeżeli się upewnisz, że mamy na to czas to spoko, ale w przeciwnym wypadku jest do wyrzucenia.